app/services/editions/db/IssueQueries.scala

```diff
@@ -223,17 +226,50 @@ trait IssueQueries {
       .apply()
   }
 
-  def publishIssue(issueId: String, user: User, now: OffsetDateTime) = DB localTx { implicit session =>
+  def publishIssue(issueId: String, user: User, now: OffsetDateTime): String = DB localTx { implicit session =>
     val userName = user.firstName + " " + user.lastName
     val truncatedNow = EditionsDB.truncateDateTime(now)
```

Comment:
- in `def publishIssue(issueId: String, user: User, now: OffsetDateTime): String` can we type alias String to `VersionId` ?

like:

`def publishIssue(issueId: String, user: User, now: OffsetDateTime): VersionId` 

it is not clear that `publishIssue` will generate and return the id

- Or generate versionId outside and pass to `publishIssue`
- i am not sure if `publishIssue` is correct function name it `createIssueVersion` or `initialiseIssueVersion` and publishing is done by downstream stuff and function that calls this one `EditionsPublishing.publish`